### ISSUE TITLE
Refresh list of EMS events collected by Harvest

### ISSUE THREAD

**Issue by cgrinds** [NetApp/harvest#1523]

**Comment by cgrinds** [comment]
Target for mid Dec or Jan 2023

**Comment by cgrinds** [comment]
Fixed by #1988

**Comment by cgrinds** [comment]
Verified on bcd22289

```bash
git switch release/23.02.0
cat conf/ems/9.6.0/ems.yaml | dasel -r yml -w json | jq '.events[].name' | wc -l
      60
git switch -
Switched to branch 'release/23.05.0'
Your branch is up to date with 'origin/release/23.05.0'.
cat conf/ems/9.6.0/ems.yaml | dasel -r yml -w json | jq '.events[].name' | wc -l
      98
```
